plugins/grid-bookmark/src/GridBookmarkWidget/components/BookmarkGrid.tsx

```diff
@@ -67,6 +54,9 @@ const BookmarkGrid = observer(function ({
       }
     })
 
+  // reset selections if bookmarked regions change
+  // needed especially if bookmarked regions are deleted, then
+  // clear selection model
```

Comment:
comment reason for clearing the row selection model on changes to bookmarkedRegions.length